src/common/blockchain.sh

```diff
@@ -1,3 +1,201 @@
 #!/usr/bin/env bash
```

Comment:
@jt-nti : Now that these functions have been moved here, would this break anything on the composer side?

@yorhodes @sanjayravindraibm, Watch for the answer to the question above.

Reply:
Not too worried about adding the new functions in the new scripts for this pull request, since the old scripts should remain unchanged for now. If there are any issues, they would need addressing when adding composer support.